spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ResourceProperties.java

```diff
@@ -53,4 +56,91 @@ public void setAddMappings(boolean addMappings) {
 		this.addMappings = addMappings;
 	}
 
+	public Chain getChain() {
+		return chain;
+	}
+
+	/**
+	 * Configuration for the Spring Resource Handling chain
+	 */
+	public static class Chain {
+
+		/**
+		 * Enable the Spring Resource Handling chain
```

Comment:
key properties must end with a dot. See [the documentation for more details](http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#configuration-metadata)